### ISSUE TITLE
Group Lasso regularization

### ISSUE THREAD

**Issue by hello072** [markdtw/condensenet-tensorflow#3]
Thank you for sharing!
Is Group Lasso regularization implemented in this repo?

**Comment by markdtw** [comment]
Yeah about that... Group Lasso regularization is not yet implemented :( 
Any help is appreciated!
